massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -378,13 +378,15 @@ impl ProtocolWorker {
             }
             ProtocolCommand::WishlistDelta { new, remove } => {
                 massa_trace!("protocol.protocol_worker.process_command.wishlist_delta.begin", { "new": new, "remove": remove });
-                self.remove_asked_blocks_of_node(remove)?;
```

Comment:
I think this should still be there because even if it doesn't remove from block_wislist anymore it remove from the asks made to the connected nodes to avoid re-ask etc

Reply:
Makes sense, re-added.